src/debugger/rnDebugSession.ts

```diff
@@ -27,18 +27,22 @@ export interface IAttachRequestArgs extends DebugProtocol.AttachRequestArguments
     port: number;
     url?: string;
     address?: string;
+    trace?: string;
 }
 
 export interface ILaunchRequestArgs extends DebugProtocol.LaunchRequestArguments, IAttachRequestArgs { }
 
 export class RNDebugSession extends LoggingDebugSession {
 
+    private readonly CDP_PROXY_PORT = 13602;
```

Comment:
Why not random?

Reply:
Added generation of random CDP Proxy port number.